### DISCUSSION TITLE
LaTeX Support

### DISCUSSION THREAD

**Discussion by dhairyaj** [Documentive/TemplateBuddy#101] [Ideas]
### Proposal
- Add a feature to create and download resumes in LaTeX format.
- This can be extended to create research papers and mostly, research papers are written in LaTeX format which provides a lot of freedom for styling.
- Adding LaTeX support might also be a better option to download resumes and research papers easier.

**Comment by frankhart2018** [comment]
This is our current approach for exporting resumes. Probably make an issue for this, closing this discussion.
